wagtailmedia/utils.py

```diff
@@ -0,0 +1,13 @@
+from wagtail import VERSION as WAGTAIL_VERSION
+
+if WAGTAIL_VERSION < (2, 5):
+    from wagtail.utils.pagination import paginate
+else:
+    from django.core.paginator import Paginator
+
+    DEFAULT_PAGE_KEY = 'p'
+
+    def paginate(request, items, page_key=DEFAULT_PAGE_KEY, per_page=20):
+        paginator = Paginator(items, per_page)
+        page = paginator.get_page(request.GET.get(page_key))
+        return paginator, page
```

Comment:
This is now merged in `master` so can be removed from the PR.

Reply:
How do you suggest to remove already merged code from PR? Create a new PR or add a commit that removes this code?

Reply:
You shouldn’t need to make a new PR. First pull the latest `master` from this repository to your fork’s `master`, then here are the best two options, from your `chooser-upload-form`:

- `git rebase master` to rebase your commits as if they had been done on top of the latest master. This will either prompt you to fix the conflicts due to this code being added twice (once in master, once in your branch) – or automagically figure out that there is no need for the same change twice (unlikely here I think)
- `git merge master` to create a merge commit of the new master into your branch. This merge commit will be removing that code.

It’s an important part of making pull requests to keep them up-to-date with a project’s changes, so worth your time learning this IMHO, but if this goes over your head I’m also happy to do it myself once I get back to your PR.

Reply:
I didn't have so much experience with rebase/merge commands. Thanks for the tips! ;)
I'll try to update PR as requested.

Reply:
I've went for `git rebase master` path but I'm unsure if that was the cleanest way. Seems that merges from updated master are also appearing.
Is it expected? If not then sorry for the noise. I can try to add a new PR based on updated codebase of wagtailmedia's master if you want.

Reply:
Hmm yes, it looks like that worked and the conflicts are gone, but now your PR contains changes that are unrelated to it.

I’ve tried to do it myself with `git rebase -i master`, which is slightly more tricky to use. Here is the result for me, with your three commits: https://github.com/thibaudcolas/wagtailmedia/tree/teixas-chooser-upload-form.

Reply:
Right, all commits that were responsible for master merge are assigned to my PR. Again sorry for the noise!
Maybe I should really go for a new branch and do another PR based on current master and definitely close this one.
Since there are only 3-4 files that need modifications so this wont be so hard. What do you think?

Reply:
Yep, that works too!

Reply:
Fixed in #97